Documentation/HowToUseReleaseDrafter.md

```diff
@@ -0,0 +1,69 @@
+# GitHub release configuration
+The Release Drafter tool uses GitHub actions inputs to control GitHub release elements (tags, release name, draft/full release status, etc.); to control changelog level elements (text format, style, label text, etc.), see [Changelog text and style configuration](#changelog-text-and-style-configuration).  
+
+For all available action inputs, see the reference: [Action Inputs](https://github.com/release-drafter/release-drafter#action-inputs)
+
+
+## How to control releases
+It is possible to have the tool output a draft release version to verify the output is acceptable; it may be helpful to have a workflow input variable to control this behavior. Note that any existing draft releases with the same tag version will be replaced by the newly created draft (that is, automatic cleanup of older drafts).
+
+# Changelog text and style configuration
+The Release Drafter tool uses a yaml config file to control the style of output. This config controls changelog text level elements; to control GitHub release level elements (tags, release name, draft/full release status, etc.), see [GitHub release configuration](#github-release-configuration).
+
+By default, the Release Drafter action looks at the repository's ~/.github/ directory (NOT the same level as the workflow itself; one level above) for a file with the name `release-drafter.yml`. If a different filename is desired, an argurment can be added to the `with` parameters (other `with` parameters not included in example snippet for brevity):  
```

Comment:
Looking to see if we can simplify this one:

By default, the Release drafter action looks for its configuration at `.github/release-drafter.yml`. To use a different config filename, specify the `config-name` as in the example below:

Reply:
Thanks, I think that your suggested wording is much more concise. I did want to emphasize that the config is in a separate directory from the workflow file itself though; what do you think about:

By default, the Release Drafter action looks for its configuration at `.github/release-drafter.yml` (note this is one level up from the action's own `workflows` directory). To use a different config filename, specify the `config-name` as in the example below: